test-clients/src/main/java/com/hedera/services/bdd/spec/queries/meta/HapiGetTxnRecord.java

```diff
@@ -286,6 +293,24 @@ private void assertPriority(HapiApiSpec spec, TransactionRecord actualRecord) th
 		expectedDebits.ifPresent(debits -> assertEquals(debits, asDebits(actualRecord.getTransferList())));
 	}
 
+	private void assertChildRecords(HapiApiSpec spec, List<TransactionRecord> actualRecords) throws Throwable {
+		if (childRecordsExpectations.isPresent()) {
+			final var expectedChildRecords = childRecordsExpectations.get();
+
+			assertEquals(expectedChildRecords.size(), actualRecords.size(), "Expected and actual child records should" +
+					" be the same size.");
```

Comment:
```suggestion
```suggestion
assertEquals(expectedChildRecords.size(), actualRecords.size(), 
					String.format("Expected %d child records, got %d", expectedChildRecords.size(), actualRecords.size()));
```
```